TWLight/users/templates/users/collection_tile.html

```diff
@@ -68,23 +68,23 @@
       <hr>
     {% endif %}
     <i class="fa fa-info-circle" aria-hidden="true" title="Partner description page"></i> <strong><a href="{% url 'partners:detail' each_authorization.partner.pk %}">{{ each_authorization.partner }}</a></strong>
-    {% if each_authorization.date_expires %}
-      {% if each_authorization.about_to_expire or not each_authorization.is_valid %}
-        {% if each_authorization.latest_app and not each_authorization.open_app %}
-          {% comment %} Translators: Hovertext for when renewals for a partner is not available and the renew button is disabled. {% endcomment %}
-          <span class="pull-right" {% if not each_authorization.partner.renewals_available %}title="{% trans 'Renewals are not available for this partner' %}"{% endif %}>
-            <a href="{% url 'applications:renew' each_authorization.latest_app.pk %}" class="btn btn-primary pull-right {% if not each_authorization.partner.renewals_available %}disabled{% endif %}" role="button">
-              {% comment %} Translators: Labels the button that links to the renewal url. {% endcomment %}
-              {% trans 'Renew' %}
-            </a>
-          </span>
-          <div class="clearfix"></div>
-        {% endif %}
-      {% endif %}
-      {% if each_authorization.open_app %}
-        {% comment %} Translators: Labels the button that links to the application page itself if the applications is already renewed. {% endcomment %}
-        <a href="{% url 'applications:evaluate' each_authorization.open_app.pk %}" class="btn btn-primary pull-right" role="button">{% trans 'View application' %}</a><div class="clearfix"></div>
+    {% if each_authorization.date_expires and each_authorization.about_to_expire or not each_authorization.is_valid or not each_authorization.partner.authorization_method == each_authorization.partner.PROXY and not each_authorization.partner.authorization_method == each_authorization.partner.BUNDLE %}
```

Comment:
This is a _very_ long line - can we simplify it via the model properties at all? For example, could `about_to_expire` include a check for `date_expires`?

Reply:
Hmm https://github.com/WikipediaLibrary/TWLight/pull/302/commits/a55e3e5d64ee65392685d81c7ce08dc3be6b88ed#diff-fb728e27df603ec50dac4ccfdb694b8fL513 :)

Reply:
happy to add the model back in of course

Reply:
Could you elaborate? We removed `is_expired` because we have the opposite function `is_valid`, right?

And actually i just took a quick look and `about_to_expire` already includes `if authorization.date_expires`, so that's an easy win in terms of reducing code here.

Reply:
sorry, I was thinking of a different thing. Didn't realise I'd already had a check in `about_to_expire` for `date_expires` – you're right, will remove the redundant check.